src/ARMInterpreter_ALU.cpp

```diff
@@ -661,7 +661,9 @@ void A_MOV_REG_LSL_IMM_DBG(ARM* cpu)
         (cpu->NextInstr[0] & 0xFF000000) == 0xEA000000 && // branch
         (cpu->NextInstr[1] & 0xFFFF) == 0x6464)
     {
-        u32 addr = cpu->R[15] + 2;
+        u32 addr = cpu->R[15] + 4; // Skip 2nd ID and flags
```

Comment:
GBA Tek says that the next 2 bytes after the 2nd ID are flags, but it seems they don't have any purpose yet, they are only _reserved for flags_. So there shouldn't be any problem skipping them. Originally, `NocashPrint` had the `+2` that skipped the flags; but since I needed to use `NocashPrint` I moved the `+2` _outside_ of the function (to lines 664 and 1532).

Here is the link to the GBA Tek docs regarding messages: [Debug Messages](http://problemkaputt.de/gbatek.htm#debugmessages)